server/test/suite/userManagement.test.mjs

```diff
@@ -67,9 +67,38 @@ test(`userManagement.getWsForUser works properly`, () => {
 
```

Comment:
Please see my other comments. We should be clearing user state between tests. We should also be just using "true" and "false" instead of setting an "expectedResult" variable each time.

This makes the tests easier to read and more stable in order to expand upon them later.

Reply:
Taken care